server/repository/src/migrations/v2_00_00/mod.rs

```diff
@@ -43,11 +43,15 @@ impl Migration for V2_00_00 {
         user_change_last_synced_to_optional::migrate(connection)?;
         inventory_adjustment_logtype::migrate(connection)?;
         report_views::migrate(connection)?;
-        requisition_line_add_item_name::migrate(connection)?;
+        run_without_change_log_updates(connection, || {
```

Comment:
Can't have one run_without_change_log and put both requisition_line and stocktake in here?

Reply:
I was 50/50 on it, I considered it easier to wrap each one than look into if there was any execution ordering issues if I moved them together. It does slightly hurt me they're not together but no one will look at this code in a few months surely 😁.